boost-maven-plugin/src/main/java/io/openliberty/boost/utils/MavenProjectUtil.java

```diff
@@ -34,5 +34,9 @@ public static String findSpringBootVersion(MavenProject project) {
 
         return version;
     }
+    
+    public static boolean isValid(String s) {
```

Comment:
Using this method on various strings makes it look like you are validating it is a valid spring boot version, or spring boot classifier etc. I suggest renaming from isValid to isNonNullAndNonEmpty or something like that.

Reply:
Sure, this can be changed.